runners/direct-java/build.gradle

```diff
@@ -28,8 +30,14 @@ description = "Apache Beam :: Runners :: Direct Java"
  * TODO: Swap to generating test artifacts which we can then rely on instead of
  * the test outputs directly.
  */
-evaluationDependsOn(":runners:core-java")
 evaluationDependsOn(":model:fn-execution")
+evaluationDependsOn(":runners:core-construction-java")
```

Comment:
Unfortunately, we still require `evaluationDependsOn` here because we're directly accessing test class directories to emulate `dependenciesToScan`. I have not yet discovered a way around this.

Reply:
Good point